end_to_end/tpu/mixtral/8x7b/1_test_mixtral.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This file, combined with step 2 in the same directory, demonstrates converting a Mixtral checkpoint from Mistral AI.
+# This step is tested nightly on an ordinary CPU VM.
```

Comment:
Nit: Both steps are run together as a nightly test. The first step converts the Mistral pytorch checkpoint to MaxText (orbax) format using only a CPU VM, whereas the second step uses the checkpoint to run inference and fine-tuning on a TPU.

Reply:
SG! Will update.